conjur/api/api.py

```diff
@@ -221,6 +225,28 @@ def get_variables(self, *variable_ids) -> dict:
 
         return remapped_keys_dict
 
+    def create_token(self, create_token_data: CreateTokenData) -> requests.Response:
+        """
+        This method is used to create token/s for hosts with restrictions.
+        """
+        if create_token_data is None:
+            raise MissingRequiredParameterException('create_token_data')
+        params = {}
+        params.update(self._default_params)
+
+        token_params = {}
+        for attr, value in create_token_data.__dict__.items():
+            if value is not None:
+                token_params[attr] = value
+
+        return invoke_endpoint(HttpVerb.POST,
+                               ConjurEndpoint.HOST_FACTORY_TOKENS,
+                               params,
+                               parse.urlencode(token_params),
+                               api_token=self.api_token,
+                               ssl_verify=self._ssl_verify,
+                               headers={'Content-Type': 'application/x-www-form-urlencoded'})
```

Comment:
can we return a JSON or a string instead of the response itself? Would be clearer

Reply:
The documentation of the logic class
`HostFactoryLogic

    This class holds the business logic for executing and manipulating
    returned data`

Reply:
i dont follow

Reply:
@sigalsax The logic responsible to manipulate the response